### PR TITLE
Add package_ncurses_6_0.

### PR DESCRIPTION
**Pull request by blankenberg**

### CODE REVIEW

File: packages/package_ncurses_6_0/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="ncurses" version="6.0">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">ftp://invisible-island.net/ncurses/current/ncurses-6.0-20150725.tgz</action>
```
Comment (erasche) [inline packages/package_ncurses_6_0/tool_dependencies.xml]: same request, would you mind adding an sha256sum to this please

**Comment by erasche** [comment]
LGTM

File: packages/package_ncurses_6_0/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                <action type="shell_command">
+                    ./configure --prefix $INSTALL_DIR --with-shared --enable-symlinks
+                </action>
+                <action type="shell_command">make</action>
+                <action type="shell_command">make install</action>
```
Comment (bgruening) [inline packages/package_ncurses_6_0/tool_dependencies.xml]: We can use `<action type="autoconf">--with-shared --enable-symlinks</action>` here. Just as information.

**Comment by bgruening** [comment]
LGTM!

**Comment by davebx** [comment]
:+1:

**Pull request merged**